sys-fs/tmsu/tmsu-0.7.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/oniony/TMSU/"
+DESCRIPTION="Files tagger and virtual tag-based filesystem"
+HOMEPAGE="https://github.com/oniony/TMSU/wiki"
+SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```

Comment:
`RESTRICT="!test? ( test )"`.

Reply:
huh, what does this syntax mean? if test is disabled disable test?

Reply:
yes

Reply:
why is this needed for this package specifically?

Reply:
o, nvm, I think I understand what this does, courtesy of iamben in IRC :)